app/src/main/java/com/brainwallet/wallet/BRPeerManager.java

```diff
@@ -61,6 +61,20 @@ public static BRPeerManager getInstance() {
      * int (*networkIsReachable)(void *info))
      */
 
+    public static void txStatusUpdate() {
```

Comment:
mistakenly moved some critical methods into BRPeerManager and re-added them in a different order